src/Nixfmt/Pretty.hs

```diff
@@ -0,0 +1,141 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE LambdaCase        #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Nixfmt.Pretty where
+
+import           Data.List     hiding (break, group)
+import           Data.Text     (Text, pack)
+import qualified Data.Text     as Text
+import           Nixfmt.Predoc
+import           Nixfmt.Types
+import           Prelude       hiding (break)
+
+instance Pretty Trivium where
+    pretty EmptyLine            = emptyline
+    pretty (LineComment lc)     = text "#" <> pretty lc <> hardline
+    pretty (BlockComment bc)
+        = text "/*" <>
+          hcat (intersperse hardline $ map text bc) <>
```

Comment:
In terms of formatting, the alignment, I’m not sure if it is in our style-guide yet, but we (I) typically prefer operator-first formatting.
```suggestion
       <> hcat (intersperse hardline $ map text bc)
```

Reply:
(Ugh, that’s just an example with one line – apparently GitHub does not support multi-line suggestions.)

Reply:
I wish a formatter would do this for me :p. I encounter a lot of cases in Haskell where I don't know what the right way is to format something. I'll fix this when I'm too tired to do more useful stuff sometime. Can you give me the reasoning for putting operators first, since it may apply to nixfmt as well?